.dockerignore

```diff
@@ -30,18 +39,8 @@ config.log
 .sconsign.dblite
 .daos_server.active.yml
 autodocs
-utils/rpms/_topdir
```

Comment:
We really would not want the contents of this dir included in the context.  Why remove it here?

Reply:
Rather than include the whole utils/ dir we now exclude (or don't include) the entire directory and just include the subdirs we want so this is now excluded automatically.

Reply:
I see.